include/swift/IDE/SyntaxModel.h

```diff
@@ -146,6 +147,7 @@ struct SyntaxStructureNode {
     case SyntaxStructureKind::StaticVariable:
     case SyntaxStructureKind::ClassVariable:
     case SyntaxStructureKind::Parameter:
+    case SyntaxStructureKind::Subscript:
```

Comment:
the syntax kinds of `get` and `set` inside `subscript`s become `identifier` instead of `keyword`